src/main/java/com/prgrms/amabnb/room/api/RoomApiController.java

```diff
@@ -0,0 +1,27 @@
+package com.prgrms.amabnb.room.api;
+
+import com.prgrms.amabnb.room.entity.dto.request.CreateRoomRequest;
+import com.prgrms.amabnb.room.service.CreateRoomService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import javax.validation.Valid;
+
+@RestController
+@RequestMapping("/rooms")
+@RequiredArgsConstructor
+public class RoomApiController {
+
+    private final CreateRoomService createRoomService;
+
+    @PostMapping
+    public ResponseEntity createRoom(@Valid @RequestBody CreateRoomRequest createRoomRequest) {
```

Comment:
```suggestion
    public ResponseEntity<Void> createRoom(@Valid @RequestBody CreateRoomRequest createRoomRequest) {
```
로우 타입은 피하는 게 좋아보입니다!

Reply:
넵!!!!